cni/network/network.go

```diff
@@ -276,6 +307,8 @@ func (plugin *netPlugin) Add(args *cniSkel.CmdArgs) error {
 		return err
 	}
 
+	plugin.report.ContainerName = k8sPodName + ":" + k8sNamespace
```

Comment:
what's this line doing / needed?

Reply:
yes this is for filtering logs based on container name